test/apiTests.js

```diff
@@ -0,0 +1,565 @@
+import chai from 'chai';
+import request from 'supertest';
+import 'babel-polyfill';
+
+import app from './../app';
+import database from './../server/src/database';
+import testData from './testData';
+
+const expect = chai.expect;
+
+describe('Given an API request', () => {
+  before(async () => {
+    await database.connection.sync({ force: true });
+  });
+
+  describe('When the request contains appropriate user registration data', () => {
+    it('Then the user is registered', async () => {
```

Comment:
Try and improve your test descriptions....something like it('should ....') would be better